js/markupBlog.js

```diff
@@ -0,0 +1,328 @@
+/** @format */
+const markupTitle = (value, borderClass) => {
+  let div = document.createElement("div");
+  let border = document.createElement("div");
+  let h2 = document.createElement("h2");
+
+  h2.className = "title";
+  div.className = "title_box";
+  border.className = borderClass;
+
+  h2.textContent = value;
+
+  div.append(h2);
+  div.append(border);
+
+  return div;
+};
+
+const createButton = (classBtn) => {
```

Comment:
this function has a duplicate too, in `markupIndex.js` and `markupPost.js` files. Please move it to a separate file to avoid duplicate logic.